### PR TITLE
Add app.py

### PR DESCRIPTION
**Pull request by MatMoore**
Now that the db stuff is merged, I thought a logical next step is to start fleshing out the api part.

This adds app.py as an entry point to the application. At the moment this has no endpoints defined, so it just serves a 404 response, and minimal API documentation, but this should make it easier to start adding stuff.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,22 @@ The intend of this project is to create an API (possibly in **Python**) which wi
...
+
+Then run the app with:
+
+```
+uvicorn covidapi.app:app --reload
+```
+
+The API will be served at [http://localhost:8000/](http://localhost:8000/)
+
+The API docs are served at [http://localhost:8000/docs](http://localhost:8000/docs)
```
Comment (andreagrandi) [inline README.md:27]: Thanks 💯 for improving the documentation!

File: covidapi/app.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from fastapi import FastAPI
+
+app = FastAPI()
```
Comment (andreagrandi) [inline covidapi/app.py:3]: Yes! That's the idea. I would like to keep the API views in the subfolder (we can discuss the naming in the separate issue #5 ) and only import the routes like this:

```
from fastapi import FastAPI
from api.views import dailyreport_controller


app = FastAPI()
app.include_router(dailyreport_controller.router)
```

Comment (andreagrandi) [reply covidapi/app.py:3]: p.s: just please add the missing line ad the end of file and we can start merging this.

**Review by andreagrandi — APPROVED** [review]

File: covidapi/app.py (hunk 2 above)
Comment (MatMoore) [reply covidapi/app.py:3]: oops, missed this comment, I will raise another PR...

**Pull request merged**